docs/assets/js/index.js

```diff
@@ -81,7 +81,7 @@ $window.resize(checkIfSidebarFits);
 checkIfSidebarFits();
 
 if (location.pathname !== '/') {
-  $('.main-content h2, .main-content h3, .main-content h4').each(function () {
+  $('.main-content h2, .main-content h3').each(function () {
```

Comment:
why?

Reply:
Because it was showing too many subsections and tree branches in the right hand side, making it harder to skim through the high-level sections.